distribution/docs/src/main/resources/_contents/_processing-framework/processing-framework-intro-contents.adoc

```diff
@@ -0,0 +1,172 @@
+|                               +-------------------------------------------------+             |
+|                               |Metacard getMetacard()                           |             |
+|                               |                                                 |             |
+|                               |                                                 |             |
+|                               +-------------------------------------------------+             |
+|                                                         ^                                     |
+|                                                         |                                     |
+|                                 +-----------------------+---------------------+               |
+|                                 |                                             |               |
+|             +---------------------------------------+                         |               |
+|             |c369    <ProcessResourceItem>          |                         |               |
+|             +---------------------------------------+                         |               |
+|             |ProcessResource getProcessResource()   |                         |               |
+|             |Boolean isMetacardModified()           |                         |               |
+|             |                                       |                         |               |
+|             +---------------------------------------+                         |               |
+|                                 ^                                             |               |
+|                                 |                                             |               |
+|               +-----------------+-------------+                               |               |
+|               |                               |                               |               |
+|+-----------------------------+ +-----------------------------+ +-----------------------------+|
+||c369 <ProcessCreateItem>     | |c369 <ProcessUpdateItem>     | |c369 <ProcessDeleteItem>     ||
+|+-----------------------------+ +-----------------------------+ +-----------------------------+|
+||                             | |Metacard getOldMetacard()    | |                             ||
+||                             | |                             | |                             ||
+||                             | |                             | |                             ||
+|+-----------------------------+ +-----------------------------+ +-----------------------------+|
++-----------------------------------------------------------------------------------------------+
+....
+
+==== ProcessingFramework
+
+The `ProcessingFramework` is responsible for processing incoming ``ProcessRequest``s that contain a `ProcessItem`. A `ProcessingFramework` should never block. It receives
```

Comment:
"``ProcessRequest``s that contain a `ProcessItem`" -> "``ProcessRequest``s that contain a list of `ProcessItem`s"